### DISCUSSION TITLE
Strange behaviour of /api/v2/usergroups endpoint

### DISCUSSION THREAD

**Discussion by mihau1211** [apache/streampipes#2448] [Dev] [answered]
Hi,
I'm facing a strange issue related to Group management and using /api/v2/usergroups.

In the frontend part, I'm adding the group by the New User Group button, in the network I can see the 200 response status so there is no error but no group was added there. 

The same behaviour is over the Postman but before I started adding any groups, the GET of /api/v2/usergroups was "500 Internal Server Error". After adding a new group, I receive the 200 but with no data in the response, just an empty array.

The version of SP that I'm using is 0.93.0.

**Comment by dominikriemer** [comment, marked as answer]
Hi @mihau1211 sorry but this is a bug in version 0.93.0 which has already been resolved in #2292. the fix will be part of the next release.
